ACKNOWLEDGEMENTS.md

```diff
@@ -1,3 +0,0 @@
-This work has been partially funded by the European Union Seventh Framework Program (FP7/2007�2013)
```

Comment:
What's the reason for deleting this ?

Reply:
The reason is to move the acknowledgment into the README.